releases/relnote-0.7.8

```diff
@@ -0,0 +1 @@
+Changing github organisation name from wattsight to volueinsight.
```

Comment:
We need to add more info here

Reply:
Could you also please add that it is only an issue when the attempt to get token fails? I believe that detail is important to include.

Reply:
Yes I will add that to the note.